### PR TITLE
fixed login button

### PR DESCRIPTION
**Pull request by anagabriel42**

### CODE REVIEW

**Comment by anagabriel42** [comment]
@sayerwalam52

File: config/routes.rb [hunk 1]
```diff
@@ -1,6 +1,7 @@
 Rails.application.routes.draw do
   get 'material_admin/index'
   root 'material_admin#index'
-  match 'material_admin/create', :via => :post 
+
+  match '/signup' => 'material_admin#create', :via => :post
```
Comment (anagabriel42) [inline config/routes.rb]: i did this to make the form code shorter

File: app/views/material_admin/index.html.erb [hunk 2]
```diff
@@ -25,26 +25,29 @@
...
+              <i class="material-icons">add</i>
+              <span>Register</span>
+            </a>
         </div>
     </div>
 
     <!-- Register -->
     <div class="lc-block" id="l-register">
-        <%= form_for(:user, :url => {:controller => 'material_admin', :action => 'create'}) do |f| %>
+        <%= form_with scope: :user, url: '/signup', local: true do |f| %>
```
Comment (anagabriel42) [inline app/views/material_admin/index.html.erb]: you might change this back to `form_for` if it doesn't work with your code, but the url should stay the same

File: app/views/material_admin/index.html.erb [hunk 3]
```diff
@@ -58,29 +61,16 @@
                     Terms and Condition
                 </label>
             </div>
-            <%= f.submit :Reg, :class => 'btn btn-login btn-success' %>
+            <%= f.button :submit, :class => 'btn btn-login btn-success' do %>
```
Comment (anagabriel42) [inline app/views/material_admin/index.html.erb]: this is how you do a submit button to add the correct styling to it

**Comment by sayerwalam52** [comment]
I reviewed all cthe changes and i think its okay with the url and route till the time its executing it normally
u can merge it so that i can pull it and add the login functionality to it and send it back to you

**Comment by anagabriel42** [comment]
okay!

**Comment by anagabriel42** [comment]
i think you have to click approve at the bottom though

**Pull request merged**